relayer/src/link.rs

```diff
@@ -902,14 +1520,20 @@ impl Link {
                 return Ok(());
             }
 
+            // Input new events to the relay path.
             if let Ok(batch) = events_a.try_recv() {
-                self.a_to_b.relay_from_events(batch.unwrap_or_clone())?;
+                self.a_to_b.process_event_batch(batch.unwrap_or_clone())?;
             }
 
+            // Input new events to the relay path.
+            self.a_to_b.step()?;
```

Comment:
Maybe find a more suggestive name for this and rename the one above? 
`process_event_batch` -> `update_schedule`
`step` -> `execute_schedule`, or `refresh_and_execute_schedule`
Or even split to be clear:
```
self.a_to_b.update_schedule()
self.a_to_b.refresh_schedule()
self.a_to_b.execute_schedule()
```

Reply:
8ad0bf4